packages/compiler/test/formatter/scenarios/inputs/alias.tsp

```diff
@@ -1,20 +1,80 @@
 alias UnionOfType = Foo | Bar;
 
-alias UnionOfManyType = "one" | "two" | "three" | "four" | "five" | "six" | "seven" | "height" | "nine" | "ten";
+alias UnionOfManyType =
```

Comment:
that shouldn't have been formatted

Reply:
lots of files that shouild be in prettierignore for formatted here, how did you trigger it?

Reply:
It made the CI happy